test/specs/smtp_spec.rb

```diff
@@ -7,8 +7,8 @@
 BODY     = 'foo'
 
 class Jackal::Mail::Smtp
-  def send_mail(args)
-    $args = args
+  def send_mail(payload, args)
+    payload.set(:smtp_args, args)
```

Comment:
We don't want to do this because we do not want to persist args into the payload data.